### ISSUE TITLE
[Issue] Blockbench 1.15 Java Entity Models Not Working Properly

### ISSUE THREAD

**Issue by bernie-g** [JannisX11/blockbench#603]
There appears to be an issue with 1.15 Java Entity models. My modeler tried using 3.5.1 to make 1.15 entity models but when he added a texture, everything was off and did not look right: 
https://i.softwarelocker.net/0pF26J.png

Therefore, he started using Blockbench 3.4.2 and everything worked fine. He then sent me both the bbmodel file and the exported .java file (1.12). When I try to import these two files, they both have separate issues.

The bbmodel looks the same as the first screenshot I sent.
The exported .java has some extra spacing that isn't there in the .bbmodel and appears to have come from nowhere: https://i.softwarelocker.net/lHbOG2.png
Those segments are touching in the 3.4.2 model, but are spaced out in 3.5.1.

Additionally, when I export the bbmodel as a 1.15 java file, it looks like this in game: 
https://i.softwarelocker.net/O0wdBT.mp4

On another note, exporting a model made in 3.4.2 as a 1.15 model seems to occasionally duplicate cubes and put them in a seemingly random location.

https://i.softwarelocker.net/RlTTxY.png (That top part is not part of the model)

This also happens to only some entities. About 90% work fine, and a couple break for seemingly no reason.

I have attached a zip file of the two models I'm trying to implement.
[BrokenEntities.zip](https://github.com/JannisX11/blockbench/files/4549556/BrokenEntities.zip)

Both of us are using the desktop version of Blockbench on Windows.

**Comment by bernie-g** [comment]
Okay, progress update. The duplication issue with the sea angel was caused by all the cubes in the model being part of the root group and blockbench automatically putting all of the subgroups in the render function. All I had to do is comment out all of the subgroup rendering and only leave the main one:

```java
        @Override
	public void render(MatrixStack matrixStack, IVertexBuilder buffer, int packedLight, int packedOverlay, float red, float green, float blue, float alpha){
		Main.render(matrixStack, buffer, packedLight, packedOverlay);
		//Lfin.render(matrixStack, buffer, packedLight, packedOverlay);
		//Lfin2.render(matrixStack, buffer, packedLight, packedOverlay);
		//neck.render(matrixStack, buffer, packedLight, packedOverlay);
	}
```
It would probably be helpful to other modders to make it not include those in the exported file by default. 

Otherwise, I still have no ideas on the shark issue.

**Comment by JannisX11** [comment]
Fixed in 3.5.2.
